### PR TITLE
add techpreview e2e to shared resoruce operator

### PR DESCRIPTION
**Pull request by gabemontero**
to mimic the tech preview periodic at https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.10-e2e-aws-techpreview

### CODE REVIEW

**Comment by gabemontero** [comment]
the 4.11 rehearsal failure is expected ... they have those files present, but the infra is not in place yet to run CI against that branch

**Comment by gabemontero** [comment]
i believe this is ready for tag for merge @coreydaley (again the 4.11 failure is expected / OK until master branch is forked from 4.10)

File: ci-operator/config/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10.yaml [hunk 1]
```diff
@@ -61,6 +61,11 @@ tests:
     test:
     - ref: csi-driver-shared-resource-e2e
     workflow: csi-driver-shared-resource-e2e-techpreview
+- as: e2e-aws-techpreview
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```
Comment (prietyc123) [inline ci-operator/config/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10.yaml:65]: @gabemontero AFAIU adding `skip_if_only_changed` at one place (presubmits file) i.e https://github.com/openshift/release/pull/24612/files#diff-b6158c401a6d08e8c4ae2868c1a04f505b7fcbe86c21b46cb587c8310293fa93R95 should be enough, no? Correct me if I am wrong.

**Review by prietyc123 — CHANGES_REQUESTED** [review]

**Pull request merged**